packages/astro/src/build.ts

```diff
@@ -110,6 +110,9 @@ export async function build(astroConfig: AstroConfig, logging: LogOptions = defa
           scanPromises.push(
             runtime.load(url).then((result) => {
               if (result.statusCode !== 200) {
+                if (result.statusCode === 404) {
+                  throw new Error(`${buildState[id].srcPath.href}: could not find "${path.basename(url)}"`);
```

Comment:
@matthewp when I try and display a helpful error message here, this is what I end up seeing:

![Screen Shot 2021-06-14 at 16 22 29](https://user-images.githubusercontent.com/1369770/121969777-39fef080-cd32-11eb-8524-a12f07e654b9.png)

It has something to do with how this is being passed to the logger, but I couldn’t figure this out. How can we fix this?

(to recreate this, simply try and run `astro build` with an asset that doesn’t exist, such as `<script src="foo.js">`)